pkg/controller/migplan/validation.go

```diff
@@ -654,3 +665,79 @@ func containsAccessMode(modeList []kapi.PersistentVolumeAccessMode, accessMode k
 	}
 	return false
 }
+
+type NamespacePods struct {
+	Namespace string         `json:"namespace"`
+	Pods      []PodWithPhase `json:"pods"`
+}
+
+type PodWithPhase struct {
+	Name   string        `json:"name"`
+	Status kapi.PodPhase `json:"status"`
+}
+
+// Validate the pods, all should be healthy befor migration
+func (r ReconcileMigPlan) validatePods(plan *migapi.MigPlan) error {
```

Comment:
This migration should be skipped while the plan is _suspended_ (migrations are running or after the _final_ migration has succeeded).
Add:
```
if plan.Status.HasAnyCondition(Suspended) {
    plan.Status.StageCondition(FailedPodStatus)
    return nil
}
```